tapir/shifts/templates/shifts/shift_calendar_template.html

```diff
@@ -8,60 +8,78 @@
     {{ block.super }}
     <link rel="stylesheet" href="{% static 'shifts/css/shifts.css' %}">
     <script src="{% static 'shifts/ShiftFiltersManager.js' %}?version=2" defer></script>
+    <script>
+        function onFilter() {
+            let url = "{% url "shifts:calendar" %}";
+            url += "?date_from=" + document.getElementById("date-from").value;
+            url += "&date_to=" + document.getElementById("date-to").value;
+            window.location = url;
+        }
+    </script>
 {% endblock head %}
 {% block content %}
-    <div class="card-group mb-2">
-        <div class="card">
-            <h5 class="card-header">
-                {% block card_title %}
-                {% endblock card_title %}
-            </h5>
-            <div class="card-body">
-                {% block card_helptext %}
-                {% endblock card_helptext %}
-            </div>
-        </div>
-        {% shift_filters %}
+    <div class="d-flex align-items-end flex-column m-2">
+        <button class="filter-button {% tapir_button_link %}"
```

Comment:
The filter-button css class is not used here, we should remove it.

Actually it's not used anywhere, it could be removed everywhere, but let's at least not add more of it.